api/src/database/database.py

```diff
@@ -58,7 +58,7 @@ def __new__(cls, *args, **kwargs):
                     cls.instance = object.__new__(cls)
         return cls.instance
 
-    def __init__(self, echo_sql=True):
+    def __init__(self, echo_sql=False):
```

Comment:
This reduces the amount of information and noise going to the logs. In case of errors, the exceptions will still contain relevant information about the failing queries, so this change doesn't affect error debugging.